cypress/e2e/driverpage.cy.js

```diff
@@ -0,0 +1,36 @@
+describe('Driver Stats Page', () => {
+  beforeEach('', () => {
+    cy.intercept('https://v1.formula-1.api-sports.io/rankings/drivers?season=2021', {
+      fixture: 'allDrivers.json'
+    })
+      .intercept('https://v1.formula-1.api-sports.io/drivers?name=Lewis%20Hamilton', {
+        fixture: 'lewis.json'
+      })
+      .intercept('https://v1.formula-1.api-sports.io/drivers?name=Max%20Verstappen', {
+        fixture: 'max.json'
+      })
+      .intercept('https://v1.formula-1.api-sports.io/rankings/teams?season=2021', {
+        fixture: 'teams2021.json'
+      })
+      .visit('http://localhost:3000/drivers')
+  });
+
+  it('should not display a driver until one is clicked on', () => {
+    cy.get('.selected-driver').contains('Click on a driver to see their stats!');
+  });
```

Comment:
I would re-word this test to say "should see instructions for viewing stats" or see if there's a div present that you can test its length to equal zero. Something like this: 
`cy.get('.driver-container')
        .get('.driver')
        .should('have.length', 0);`

As your test is written now, you aren't really testing that no drivers are visible, just that the text is there.

Reply:
That's a great point, thank you!  I have a real blind spot when it comes to checking if things aren't showing when they're not supposed to be, so I appreciate that suggestion